docs/fides/docs/installation/configuration.md

```diff
@@ -4,23 +4,53 @@ Fidesctl supports two methods of configuration. The first is via a toml file, an
 
 By default the fidesctl CLI doesn't require a config file and will instead leverage the default values. These are very likely to be wrong however so it is recommended to always configure your settings properly.
 
+
 ## Configuration file
 
-After initializing fidesctl, a default configuration file will be generated and placed within the `.fides` directory. Here's an example of a fidesctl configuration file:
+After initializing fidesctl, a default configuration file will be generated and placed within the `.fides` directory. Here's an example of a default fidesctl configuration file and documentation for each section:
 
-```toml title="fidesctl.toml"
-[api]
-database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
-log_destination = ""
-log_level = 20
-log_serialization = ""
+=== "default fidesctl.toml"
 
-[cli]
-server_url = "http://localhost:8080"
+    ```toml
+    [api]
+    database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
+
+    # Logging
+    log_destination = ""
+    log_level = INFO
+    log_serialization = ""
+
+    [cli]
+    local_mode = False
+    server_url = "http://localhost:8080"
+
+    [user]
+    encryption_key = "test_encryption_key"
+    ```
+
+=== "API Section"
+
+    | Name | Type | Default | Description |
+    | :----: | :----: | :-------: | :-----------: |
+    | test_database_url | String | ""| If the `FIDESCTL_TEST_MODE` environment variable is set to `True`, the `test_database_url` is used instead of the `database_url` to avoid overwriting production data. |
+    | database_url | String | postgres:fidesctl@fidesctl-db:5432/fidesctl | The PostgreSQL database connection string for the fidesctl database. __NOTE__: Do not include the driver here, fidesctl will do this for you. |
```

Comment:
Let's swap these lines so it reads a little more intuitively.

```suggestion
    | database_url | String | `postgres:fidesctl@fidesctl-db:5432/fidesctl` | The PostgreSQL database connection string for the fidesctl database. **Do not** include a database driver here; fidesctl will include it automatically. |
    | test_database_url | String | ""| Used instead of the `database_url` when the `FIDESCTL_TEST_MODE` environment variable is set to `True`, to avoid overwriting production data. |
```

Reply:
done